### DISCUSSION TITLE
Vote for your tools/libraries to be ported to z/OS

### DISCUSSION THREAD

**Discussion by IgorTodorovskiIBM** [zopencommunity/meta#433] [Announcements]
We have launched another poll to gather your input on which open source tools or libraries you would like to see ported to z/OS.

* [Click here to vote in the poll](https://app.sli.do/event/iRcuo82KSVjWUsBgDrUzEb/live/polls)

Your feedback is greatly appreciated and will help with our prioritization efforts going forward!

Check out the previous poll here: https://github.com/orgs/ZOSOpenTools/discussions/298

**Comment by IgorTodorovskiIBM** [comment]
Here's a look at the poll results (44 participants) as of October 2: 
[poll-results](https://github.com/ZOSOpenTools/meta/files/12787255/poll-results-2.pdf)

Top 4 are Rust, Coreutils (rest of the tools), Rsync and zsh

**Reply by MikeFultonDev** [reply]
I get access denied

**Reply by IgorTodorovskiIBM** [reply]
Turns out the link expired. I've attached the results.

**Reply by v1gnesh** [reply]
Terraform does build, however, the dependencies situation is messy (at least for me) with Go projects w.r.t sending upstream PRs, getting higher upstream releases included in a downstream project, etc.

NATS (a connectivity/messaging product) builds and skeletal zOS support was added in a recent release.
Will need your help to get a port assigned for NATS to test it out :)
